src/test/java/teammates/test/cases/ui/browsertests/AdminHomePageUiTest.java

```diff
@@ -144,7 +145,8 @@ private void testCreateInstructorAction() throws Exception {
         //verify sample course is accessible for newly joined instructor as an instructor
         
         ______TS("new instructor can see sample course in homepage");
-        InstructorHomePage instructorHomePage = AppPage.getNewPageInstance(browser, InstructorHomePage.class);  
+        InstructorHomePage instructorHomePage = AppPage.getNewPageInstance(browser, InstructorHomePage.class);
+        ThreadHelper.waitFor(2000);
```

Comment:
An alternative will be placing {*} in the place where the (0 / 9) is expected. Will that be better?